json_test.py

```diff
@@ -1238,38 +1238,6 @@ def test_simple_schema(self):
         """
         run_func_docstring(tester=self, test_func=self.test_simple_schema)
 
-    def test_pkey_requirement(self):
```

Comment:
Can you add to the commit message which unit tests cover it, for clarity? Thanks

Reply:
I saw you added respective unit test in the cassandra repo. We should either backport that test to previous versions too or we should keep this one up to 5.0.
Something like this - `@since("2.2", max_version="4.X")`. and port the cassandra test only to 5.0 maybe?
We can say up to 5.0 but then it won't test probably patch versions of 5.0